pkg/accountmanager/poller.go

```diff
@@ -145,7 +145,7 @@ func (p *accountPoller) doAccountPolling() {
 		p.updateAccountStatus(p.cloudInterface)
 	}()
 
-	cloudInventory, err := p.cloudInterface.GetCloudInventory(p.accountNamespacedName)
+	cloudInventory, err := p.cloudInterface.GetAccountCloudInventory(p.accountNamespacedName)
```

Comment:
I changed it because service config also has func `GetCloudInventory`, but thats just for a single service config. Since account config can now have multiple service configs and this func combines all service configs' inventory, I want to change this func name to differentiate the two.

Don't quite like ByAccount also, since we are already got a specific account config, how about `GetCloudInventoryForAccount`?